shivyc/main.py

```diff
@@ -118,6 +122,12 @@ def get_arguments():
     parser.add_argument("-z-reg-alloc-perf",
                         help="display register allocator performance info",
                         dest="show_reg_alloc_perf", action="store_true")
+    parser.add_argument(
+        "-o",
+        nargs="?",
```

Comment:
If I am not mistaken, as written this will accept the command `shivyc -o` without warning or error. However, it appears GCC prints an error here, so we should do the same.

Reply:
The shivyc will print "error: the following arguments are required: files" with -o option without any arguments.If we have to change the argparse warning/error message,maybe we have to customize the argument parser?

Reply:
Fix this on new pull request #101 .Code review is required.